src/lib.rs

```diff
@@ -687,3 +687,19 @@ impl BlitPipeline {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    fn assert_is_send<T: Send>() {}
+
+    #[test]
+    fn check_scene_is_send() {
+        assert_is_send::<super::Scene>();
+    }
+
+    #[test]
```

Comment:
The `#[test]` attribute is a bit weird here. I think a `const _: () = {}` block is more idiomatic for this kind of check.

Reply:
I've seen it done this way in Bevy and some other crates in the past.

I did it this way partly so that there is an entry in the test output indicating which checks it performed, so that I would know in this case that it actually checked the `Renderer` and not a dumb mistake in the `cfg` check.

Reply:
I still think I'd prefer it - now that we know that the check is working as expected.

Reply:
If there's a precise change you want to see here with material benefits, please provide it. This is close to how various other crates do it (bevy, static assertions crate, etc).

Reply:
I thought my suggestion is exactly what the static-assertions crate implements?

Reply:
Ends up it is a regular dependency.